elements.py

```diff
@@ -21,15 +21,17 @@ def display(self, game):
         # Collision detection test
         if element_rect.colliderect(car1_rect) or element_rect.colliderect(car2_rect):
             if self.element in (game.RED_SQUARE, game.BLUE_SQUARE):
-                game.collision = 1
+                game.collision = True
                 game.hit.play(0)  # Play sound
             if self.element in (game.RED_CIRCLE, game.BLUE_CIRCLE):
                 game.scoresound.play(0)  # Play sound
                 game.score += 1
                 game.objectlist.remove(self)
 
+        # This makes the objects move down the y-axis,
+        # changing it will affect perceived speed of the cars
         if game.move:
-            self.y_to_draw += 5
+            self.y_to_draw += game.background_height * 0.0065
```

Comment:
How'd `0.0065` come about?

Reply:
Around this value the speed of the cars feels ideal. Do you suggest something else?

Reply:
Okay.